src/107-Arduino-TMF8801.h

```diff
@@ -55,6 +55,11 @@ class ArduinoTMF8801 : public LengthSensorBase
 
   bool begin(uint8_t const measurement_period_ms);
 
+  uint32_t read_serial_number(void);
+  void set_gpio(TMF8801::GPIO const gpio);
+  void clr_gpio(TMF8801::GPIO const gpio);
+  void stop_continuous_measurement();
+  void change_i2c_address(uint8_t new_address);
```

Comment:
```suggestion
  void change_i2c_address(uint8_t const new_address);
```
Const-correctness 4tw ;) Also change in function definition.